mmv1/third_party/terraform/utils/iam.go.erb

```diff
@@ -475,6 +484,12 @@ func IamWithGAResourceDeprecation() func (s *IamSettings) {
 	<% end -%>
 }
 
+func IamWithBatching(enableBatching bool) func(s *IamSettings) {
```

Comment:
Now that we can omit the setting to indicate batching is off, we should allow us to make this `IamWithBatching` and enable batching whenever present!  i.e.:

```
func IamWithBatching(s *IamSettings) {
    s.EnableBatching = enableBatching
}
```

That way callers just specify `IamWithBatching` instead of `IamWithBatching(IamBatchingEnabled)`

Reply:
Made the change.